data-model/SDP-data-model-templates/manuscript-object_sdp-template.json

```diff
@@ -65,48 +62,30 @@
         }
       ],
       "sources": [
-        "ARK?"
-      ]
-    }
-  ],
-  "paratexts": [
```

Comment:
A paratext could have as its scope the manuscript object (e.g., waqf or custody notes?). However, I wonder if for now we should keep these as fields in the codicological (for colophons, etc.) or textual (for rubrics, and perhaps even readers notes?) level. 

If we do ever spin them out into their own textual artifact-like object records this point might be moot.